scripts/run_docker_build.sh

```diff
@@ -13,6 +13,8 @@ channels:
 
 always_yes: true
 show_channel_urls: true
+track_features: 
+ - nomkl
```

Comment:
This just leaked out from me trying to speed up my local builds. I'll revert it. Good catch